script.js

```diff
@@ -1 +1,158 @@
 // Please implement exercise logic here
+let handle = null; // for setInterval and clearInterval
+let handle2 = null;
+let start = true;
+let prevTime = 0; // for Date.now() difference with elapsedTime
+let elapsedTime = 0;
+let firstSplit = true;
+let split = false;
+let prevSplitTime = 0;
+let elapsedSplitTime = 0;
+
+//build stopwatch elapsed time
+const stopWatch = document.createElement("div");
+stopWatch.classList.add("elapsed");
+stopWatch.innerHTML = `00 : 00 : 00 : 000`;
+document.body.appendChild(stopWatch);
+
+// build interactive buttons
+const imgContainer = document.createElement("div");
+imgContainer.classList.add("image-container");
+document.body.appendChild(imgContainer);
+const goStart = document.createElement("img");
+goStart.classList.add("img", "start");
+goStart.src = "/assets/play.png";
+imgContainer.appendChild(goStart);
+const goStop = document.createElement("img");
```

Comment:
I think you can simply name these variables as `startTimer`, `stopTimer`, or even `startButton`, `stopButton` ... etc